core/trino-main/src/main/java/io/trino/cost/ComparisonStatsCalculator.java

```diff
@@ -22,7 +22,9 @@
 import static io.trino.cost.SymbolStatsEstimate.buildFrom;
```

Comment:
I don't think it needs to be behind flag. We actually added pretty reasonable estimate.

Reply:
I would prefer to keep the flag for now and remove it after a few releases once we have validation from real workloads.

Reply:
removed flag